cmd/release-controller/sync_tags.go

```diff
@@ -215,6 +215,44 @@ func (c *Controller) ensureReleaseTagPhase(release *Release, preconditionPhases
 	return nil
 }
 
+// Update annotations for a set of tags in a release
+func (c *Controller) updateReleaseTagAnnotations(release *Release, tagAnnotations map[string]map[string]string) error {
```

Comment:
`map[string]map[string]string` is awkward and usually frowned upon.  Use an interface.